### PR TITLE
Distinguish start/end brackets and @ symbols

### PR DESCRIPTION
**Pull request by MaxGiting**
This makes blade syntax easier to pick out in large blocks of HTML and also brings it more inline with other highlighters like the Sublime Text highlighter

### CODE REVIEW

**Comment by Ingramz** [comment]
I'm not sure if semantically the `@` sign should be treated as a function. I'll look into it at work and then get back to you.

**Comment by Ingramz** [comment]
Alright, by now I am quite confident that specifically the function selector is not very appropriate here semantically. Maybe we can find something better that we can use instead.

However, I agree that with certain themes the blade syntax does not stand out and something needs to be done in order to mend that. Thank you for pointing it out.

**Comment by MaxGiting** [comment]
I've changed the `support.function.blade` to `support.constant.blade`. This is inline with the Sublime Text grammar here.
https://github.com/Medalink/laravel-blade/blob/master/laravel-blade.tmLanguage#L80-L86

Also the most definitive list I could find anywhere seems to be the Text Mate docs. I tried a few scope variations and not many highlighted the same colour as I had before. There doesn't seem to be any specific category it slotted into 100% but this is much better.
http://manual.macromates.com/en/language_grammars
I found that from this Atom Discussion
https://discuss.atom.io/t/canonical-list-of-syntax-scope-names/13430
Which suggest's it is the most accurate list

**Comment by Ingramz** [comment]
While constant isn't much better as it is regularly used for... well language specific constants (think literals like integers values, true, false, undefined, null)  - see the TextMate manual. I am not asking to change anything right now. Let me look around and see if I can find a better way of doing this.

As mentioned at the discussion board by Dan, with Atom we should be working towards more consistent usage of different grammar scope descriptors across languages, so that they could be better themed. That effectively means that TextMate/Sublime way of doing things could be flawed, so take these with a grain of salt. I'd like to point out that grammars that can be found at [TextMate's Github page](https://github.com/textmate) have generally quite well named scopes and should be preferred over third party packages.

The problem at hand is that theme developers develop themes based on grammar scopes and grammar developers name scopes after theme defined scopes - which is the chicken or the egg problem all over again. The theory (of semantically named scope) heavily conflicts with practice (where scope names are chosen on how they looked with the theme that the developer used), which is far from ideal situation.

In summary I believe the themes are more at fault than the grammar currently is, but I am not ruling out the current scope name quality in blade grammar. Would like to hear what others have to say about it too. I would also like to avoid any sort of temporary hacks because they tend to become permanent solutions after a while.

**Comment by MaxGiting** [comment]
I completely agree with you, but to the average end user I'm not sure this is a big issue. I know this may seem a temporary hack, but it will still be an improvement. It's wrong currently, but will display better with these amendments.

Not sure this is the right place to discuss changing, extending grammar scopes as a whole. It won't get the correct audience in this repo.

**Comment by Ingramz** [comment]
I agree with the very last thing you said. There are two places where this can be discussed - atom/atom issues or the discussion board. But even at those places it is rather hard to get attention of people if very few people care (eg the discussion topic that was posted before) - it needs to be served in a way that would attract as many people as possible.

**Comment by MaxGiting** [comment]
Feel free to close this pull request if you're not intending on pursuing this issue further.

**Comment by Ingramz** [comment]
I took a look at PhpStorm implementation (which uses revamped textmate themes), compared against equivalent Atom themes and found that they also color the keywords in the exact same way as we have them right now. From themes that I tried, most of them have a distinct coloring for the keywords between HTML tags, but there are still lots of themes which color them same way as well. But as discussed, quality of themes can vary a lot.

I do not think we are at fault here. For future refrence, if your theme cannot handle this case in the way you would like to, either correct it in theme or open `File -> Open Your Stylesheet` and include the following snippet.
```less
atom-text-editor::shadow .keyword.blade {
  color: red;
}
```
